numpy/lib/tests/test_function_base.py

```diff
@@ -1267,6 +1267,14 @@ def test_empty(self):
         assert_array_equal(a, np.array([0]))
         assert_array_equal(b, np.array([0, 1]))
 
+    def test_finite_range(self):
+        # Normal ranges should be fine
+        vals = np.linspace(0.0, 1.0, num=100)
+        histogram(vals, range=[0.25,0.75])
+        # non-finite values should raise AttributeError
```

Comment:
This is why (some) people sneer at comments: it wasn't updated and now contradicts the code.  I'll leave it up tp you whether to remove it entirely (the code is pretty much self explanatory, I think) or to update it to `ValueError`.